docs/2-deployments/7-multitenant-deployments.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Multi-Tenant Deployments
+---
+
+# Multi-Tenant Deployments
+
+Multi-tenant deployments are when
+
+By default, deploying a component will create it as a singleton in the environment, where it is the default instance.
```

Comment:
This section might need some small note or aside as to how this is different from horizontal scaling. (ie, we probably don't want to leave the customer confused and think they need to deploy multiple instances in order to horizontally scale).